docs/en/ingest-management/migrate-auditbeat-to-agent.asciidoc

```diff
@@ -0,0 +1,142 @@
+[[migrate-auditbeat-to-agent]]
+= Migrate from {auditbeat} to {agent}
+
+In order to migrate from using {auditbeat} to using the {agent}, there
+are some steps to take into account to be sure to have all the previous
+functionality.
+
+First of all, {auditbeat} needs to be uninstalled from the hosts and the {agent}
+will need to be deployed instead. More information about how to do this can be
+found https://www.elastic.co/guide/en/fleet/current/migrate-beats-to-agent.html[here].
+
+[discrete]
+[[compatibility]]
+== Compatibility
+
+The integrations that provide replacements for `auditd` and `file_integrity` modules,
+are only available for versions of the stack bigger or equal to `8.3`.
+
+[discrete]
+[[migrating-from-auditbeat-modules]]
+== Migrating from Auditbeat modules
+
+Depending on what Auditbeat modules you are using, they will be replaced
+by different components.
+
+[discrete]
+[[auditd-module]]
+=== https://www.elastic.co/guide/en/beats/auditbeat/current/auditbeat-module-auditd.html[Auditd]
+
+The `auditd` module has a direct replacement with the
+https://docs.elastic.co/en/integrations/auditd_manager[Auditd Manager integration]. 
+The rules and configuration can be ported to the integration. Starting from `8.4` stacks,
+this integration also provides the ability to set the `immutable` flag to the audit
+configuration.
+
+If you do not need the integration to manage the rules, there is also the
+https://docs.elastic.co/integrations/auditd[Auditd Logs] integration as an alternative,
+which will only parse logs from the audit daemon `auditd`.
```

Comment:
We should be clear that there are differences between what you get from parsing auditd logs and using Auditbeat.